tests/queries/0_stateless/01355_CSV_input_format_allow_errors.reference

```diff
@@ -0,0 +1,40 @@
+File generated:
+0,0
+1,0
+2,0
+3,0
+4,AAAAAAA
+5,0
+6,AAAAAAA
+******************
+attempt to parse w/o flags
+Return code: 117
+OK: stderr contains a message 'is not like Int64'
+******************
+attempt to parse with input_format_allow_errors_ratio=0.1
+Return code: 117
+OK: stderr contains a message 'Already have 1 errors out of 5 rows, which is 0.2'
+******************
+attempt to parse with input_format_allow_errors_ratio=0.3
+0	0
+1	0
+2	0
+3	0
+4	0
```

Comment:
FYI ill-formatted rows should be skipped, but it does not happen in this case because of [another issue](https://github.com/ClickHouse/ClickHouse/issues/7943#issuecomment-559557022)